content/log/explore.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Explore
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+## Overview 
+
+The Log explorer is an optimized view which makes it easy to do any kind of troubleshooting and data exploration.
+
+{{< img src="log/explore/explore_view_with_comments.png" alt="Explore view with comments" >}}
+
+In this view you can:
+
+* [Interact with the Time range](#time-range)
+* [Display logs into lists](#log-list)
+* [Use facets to help you in various investigations](#facets)
+* [Enter search queries](#search-bar)
+
+## Time Range
+Time range allows you to display logs on a given time period. It is symbolised by the timeline directly under the search bar. The timeline can be displayed or wrapped up with the **Show Graph** check box:
+
+{{< img src="log/explore/timeline.png" alt="Timeline" >}}
+
+Change the time periode displayed with quick ranges that are directly available:
+
+{{< img src="log/explore/timerange.png" alt="Timerange" style="width:75%;">}}
+
+## Log list
+The log list is the list of events that match the selected context, a context is defined by a [search bar](#search-bar) filter and a [time range](#time-range).
```

Comment:
> The log list is the list of events that match the selected context, a context is defined by a [search bar](#search-bar) filter and a [time range](#time-range).

The log list shows events [logs?] that match the selected context, which is defined by a [search bar] filter and a [time range].

Reply:
logs indeed.